torch/_functorch/partitioners.py

```diff
@@ -37,8 +37,8 @@
     import sympy
 
 
-AOT_PARTITIONER_DEBUG = config.debug_partitioner
-log = logging.getLogger(__name__)
+AOT_PARTITIONER_DEBUG: bool = config.debug_partitioner
+logger: logging.Logger = logging.getLogger(__name__)
```

Comment:
Let's leave this named `log` rather than `logger` since that is the convention elsewhere.

Reply:
Updated the PR with this change.